front/app/utils/actionTakingRules.ts

```diff
@@ -146,11 +146,7 @@ export const getIdeaPostingRules = ({
     const { disabled_reason, future_enabled, enabled } =
       project.attributes.action_descriptor.posting_idea;
 
-    if (
-      signedIn &&
-      (isAdmin({ data: authUser }) ||
-        isProjectModerator({ data: authUser }, project?.id))
-    ) {
```

Comment:
Needed the authUser check for canModerateProject and it's the same as signedIn, so removed signedIn